styles/main.css

```diff
@@ -3,6 +3,6 @@ h1{color: #EDF5E1;}
 h2{color: #EDF5E1;}
 a{color: #379683;}
 nav a{color: #05386B;}
-.byline{color: rgb(5,56,107,0.5);}
+.byline{color: rgba(5,56,107,0.5);}

```

Comment:
you need the `a` (for 'alpha compositing' https://en.wikipedia.org/wiki/Alpha_compositing) here to make the transparency work